diff --git a/node_modules/keyv-file/lib/index.js b/node_modules/keyv-file/lib/index.js
index ef5b351..ff3d815 100644
--- a/node_modules/keyv-file/lib/index.js
+++ b/node_modules/keyv-file/lib/index.js
@@ -69,10 +69,6 @@ class KeyvFile {
             if (!data) {
                 return defaults;
             }
-            else if (this.isExpired(data)) {
-                this.delete(key);
-                return defaults;
-            }
             else {
                 return data.value;
             }
@@ -87,9 +83,7 @@ class KeyvFile {
     keys() {
         let keys = [];
         for (const key of this._cache.keys()) {
-            if (!this.isExpired(this._cache.get(key))) {
-                keys.push(key);
-            }
+            keys.push(key);  
         }
         return keys;
     }
